content/en/docs/tasks/administer-cluster/use-cascading-deletion.md

```diff
@@ -303,7 +303,7 @@ For details, read the [documentation for your Kubernetes version](/docs/home/sup
 Run the following command:
 
 ```shell
-kubectl delete deployment nginx-deployment --cascade=orphan
+kubectl delete deployment nginx-deployment --cascade=true
```

Comment:
Why is this the right fix for the v1.25 docs? I think recommending `--cascade=orphan` is correct.

Reply:
This code block is in the version < v1.20.x tab, not in the >= v.1.20.x, please expand the code block to check.
So the flag to orphan dependents should be `--cascade=true`.
PTAL thanks.

Reply:
Ah. We should actually _drop_ the tab for v1.20 - see https://kubernetes.io/docs/home/supported-doc-versions/

Reply:
Yes, we should honor the version rule. And deletion is easy.
But at this time point, maybe we could just keep the `prior to v1.20` tab, and make the information correct.
How do you think about it?

Reply:
> maybe we could just keep the `prior to v1.20` tab, and make the information correct. How do you think about it?

This isn't a priority for me personally - I'd prefer to see the docs move directly towards the desired state by dropping that stale tab.

Reply:
OK, updated with dropping the `prior to v1.20` tab.
PTAL thanks.